src/mission_start.cpp

```diff
@@ -1627,3 +1627,82 @@ void mission_start::ranch_bartender_4(mission *miss)
 void mission_start::place_book( mission *)
 {
 }
+
+const tripoint reveal_destination( const std::string &type, bool random )
```

Comment:
`reveal_destination( , false)` still isn't being used. The flag should be removed.